sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseDataCollector.java

```diff
@@ -413,4 +451,94 @@ static CountAndDuration decodeCountAndDuration(long countAndDuration) {
             return new CountAndDuration(countAndDuration >> 44, countAndDuration & MAX_DURATION);
         }
     }
+
+    class OpenTelMetricsStorage {
+        private volatile ConcurrentHashMap<String, OpenTelMetric> metrics = new ConcurrentHashMap<>();
+        // setting most amount of openTel metrics a user can stream to this. will review if users surpass threshold.
+        private final int maxMetricsLimit = 50;
+        // setting buffer time for receiving data for a metric before declaring it inactive. (in minutes)
+        private final double metricBufferTime = 5;
```

Comment:
Can you explain what it means to declare a metric inactive and why 5 minutes was chosen? Just want to understand this in the context of live metrics specs - to my knowledge, we should be sending post requests every second

Reply:
got rid of the timestamping mechanism